spec/spec_helper.rb

```diff
@@ -94,3 +94,14 @@
   Kernel.srand config.seed
 =end
 end
+
```

Comment:
Now for tests we can just use login_user and it does our oauth for us. Cool little trick I did in monster shop. Mike didnt like it but he didnt say what was better lol

Reply:
This is very cool, nice work.